open_data_schema_map.module

```diff
@@ -62,7 +63,7 @@ class OpenDataSchemaMapException extends Exception {
  */
 function open_data_schema_map_menu() {
   $items = array();
-  $pre = 'admin/config/services/odsm';
+  $pre = OPEN_DATA_SCHEMA_MAP_ADMIN_PATH;
```

Comment:
dkinzer [10:45 AM] 
I needed to reuse  OPEN_DATA_SCHEMA_MAP_ADMIN_PATH


[10:45] 
in another file


dafeder [10:46 AM] 
ah


dkinzer [10:46 AM] 
updating $pre to use the constant was the easiest way to make the change without introducing errors while at the same time allowing me to use the value ( OPEN_DATA_SCHEMA_MAP_ADMIN_PATH)


[10:46] 
in another context


dafeder [10:46 AM] 
i didn't catch that yet


[10:47] 
using the constant in the menu function would have instroduced errors? (or adding $pre in the second instance?) obviously a small thing just looks inconsistent


dkinzer [10:48 AM] 
it’s inconsistent because they should have used a constant to begin with


[10:48] 
I use it here: https://github.com/NuCivic/open_data_schema_map/pull/95/files#diff-069cd7588c0086578da33d4998833ff6R25


[10:48] 
of course not.
Dan Feder
using the constant in the menu function would have instroduced errors? (or adding $pre in the second instance?) obviously a small thing just looks inconsistent
Posted in #unknown-channelToday at 10:47 AM


[10:49] 
I just didn’t want to refactor more than I needed to.


[10:50] 
substituting here for the defined constant introduces the least amount of change to code I did not write myself.